201-decrypt-running-linux-vm/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "value": "vmName"
+    },
+    "volumeType": {
+      "value": "All"
```

Comment:
One of your other deployment templates has "OS", "Data", and "All" as "allowedValues" for this parameter.